docs/components/file-input-field/demo/base-demo.md

```diff
@@ -1,12 +1,15 @@
 ```hbs template
-<Form::Fields::FileInput
-  @deleteLabel='Delete file'
-  @label='Label'
-  @files={{this.files}}
-  @hint='Hint text'
-  @trigger='Browse Files'
-  @onChange={{this.handleChange}}
-/>
+<div class='max-w-md'>
```

Comment:
The `file-input` is now full-width so that the list items can take up the maximum amount of space (after talking with Mary).  Due to this, I set a max-width on the demo so it doesn't look as funky.